app/static/scss/admin/navbar.scss

```diff
@@ -113,8 +149,57 @@ $w-9: 9rem;
     font-size: 2.5em;
 }
 
+@media screen and (max-width: $phone-max) {
+    nav {
+        left: (-1 * $nav-wide) !important;
+        width: $nav-wide;
+
+        .nav-item {
+            width: $nav-wide !important;
+        }
+
+        &.open {
+            left: 0 !important;
+            .ham {
+
+                .image {
+                    display: none;
+                }
+
+                .close {
+                    display: block;
+                }
+            }
+        }
+    }
+
+    .main {
+        margin-left: 0 !important;
+    }
+
+    .content {
+        width: 100%;
+        margin: 0 auto;
```

Comment:
I think you can remove `margin: 0 auto` here.  Or at the very least set it to `margin: 0`.  If the `.content` div is full width, there's no reason to use `auto` margin on the sides to center it.